core/src/main/scala/higherkindness/mu/rpc/srcgen/compendium/CompendiumClient.scala

```diff
@@ -28,62 +20,36 @@ trait CompendiumClient[F[_]] {
 
 object CompendiumClient {
 
-  def apply[F[_]]()(implicit
-      interp: InterpTrans[F],
-      clientConfig: HttpConfig,
-      F: Sync[F]
+  def apply[F[_]: Sync](
+      clientF: Client[F],
+      clientConfig: HttpConfig
   ): CompendiumClient[F] =
     new CompendiumClient[F] {
 
       val baseUrl: String          = s"http://${clientConfig.host}:${clientConfig.port}"
```

Comment:
**suggestion**: I'd parametrize the `http` part of the Url (it could be `https`).

Reply:
Or merge both in a single property called "connectionUrl" or something like that

Reply:
Yeah I would put the whole URL, including the port number, into a single property (`connectionUrl`? `serverUrl`?). Fewer things for the user to configure, and no assumptions about how they want to host the server.

Reply:
for now, it could be merged. It was split because we had, at least, two endpoints to call. 

So, does this work for both of you?
```
val versionParam = version.fold("")(v => s"?version=${v.show}")
        val connectionUrl = s"https://${clientConfig.host}:${clientConfig.port}/v0/protocol/$identifier$versionParam"
```

Reply:
Sorry, I wasn't very clear. My suggestion was to have the server's base URL (e.g. `https://devtools.mycorp.com:1234/compendium`) passed in as a single property inside `HttpConfig`. Inside this method we would construct the full URL:

```scala
val connectionUrl = s"${clientConfig.serverUrl}/v0/protocol/$identifier$versionParam"
```